MUDscape/driver.py

```diff
@@ -73,6 +82,22 @@ def send(self, message='', prompt=True):
 
         self.request.sendall(message.format(**Fore.__dict__).encode('utf-8'))
 
+    def bufferInput(self):
+        data = bytes()
+        newLine = "b\'\\r\\n\'"
```

Comment:
Uhh, this isn't how you specify byte literals.

I believe this is what you should do:

```Python
newLine = '\r\n'.encode('ascii')
```

etc.